features/list-style.yml

```diff
@@ -0,0 +1,116 @@
+name: List style
+description: The `list-style` CSS and the `list-style-image`, `list-style-position`, and `list-style-type` longhands style the marker for a list item which can be selected using the `::marker` pseudoelement.
```

Comment:
Thinking more about this, I remembered that you can use `list-style` outside of `::marker` (duh, of course, I've been doing this forever). With `::marker` being quite new (and not baseline), it feels odd to me to add it to this feature, also because it allows devs to do things that `list-style` doesn't.

My understanding is that there are 2 levels of marker customizations in CSS.
First level: `list-style`, which lets you change the type of marker, or even use an image.
Second level: if that's not enough, `::marker` lets you change the color or size of markers, independent of the list item content. And, if that's still not enough, it allows you to use your own text content as the marker.

Thoughts about splitting these features?